### PR TITLE
Tiny Template Improvements

### PR DESCRIPTION
**Pull request by juancarlospaco**
- Add GitHub Avatar Images, with responsive loading.
- Replace PNG icon with builtin CSS icon, saves 1 request.
- Logo SVG optimized, image is 95.91% smaller now, original preserved.
- Makes web background more friendly to the eyes instead of fully white, no images used.

If on Mobile it downloads a 25 x 25 px PNG else 99 x 99 px PNG for avatar.

nim-temp

:smile_cat:

### CODE REVIEW

**Comment by Yardanico** [comment]
Why random color is required though?

**Comment by juancarlospaco** [comment]
Its a very simple proc, fixed colors are boring.
Not too important tho, feel free to cherrypick, or we can hardcode a fixed color.

File: templates/pkg.tmpl [hunk 1]
```diff
@@ -48,8 +48,12 @@
     <div class="tile tile-centered divided">
       <div class="tile-content">
         <div class="tile-title">
-          <figure class="avatar avatar-xs" data-initial="" style="background-color: #5764c6;">
-          </figure>
+          <picture>
+            <!-- if Mobile 25px PNG else 99px PNG -->
+            <source srcset="https://github.com/${pkg["github_owner"].str}.png?size=99" type="image/png" media="(min-width: 767px)">
+            <img class="avatar avatar-xl" src="https://github.com/${pkg["github_owner"].str}.png?size=25" alt="${pkg["github_owner"].str}'s avatar">
```
Comment (dom96) [inline templates/pkg.tmpl]: IMO it's better to just keep the avatar small on all devices.

File: templates/base.tmpl [hunk 2]
```diff
@@ -32,7 +33,7 @@
         </section>
         <section class="navbar-section">
           <a href="/packages.xml" class="btn btn-link">
-            <img src="/img/feed-icon-14x14.png">
+            <i class="fa fa-rss"></i>
```
Comment (dom96) [inline templates/base.tmpl]: 👍

File: templates/base.tmpl [hunk 3]
```diff
@@ -1,4 +1,5 @@
 #? stdtmpl | standard
+#import random_font_color
```
Comment (dom96) [inline templates/base.tmpl]: I admire you trying to get your packages into this project, but I really don't think this should be accepted. White is perfectly fine.

File: templates/pkg.tmpl (hunk 1 above)
Comment (dom96) [reply templates/pkg.tmpl]: oh and btw <picture> is not supported AT ALL on IE.

Comment (juancarlospaco) [reply templates/pkg.tmpl]: :+1:

File: templates/base.tmpl (hunk 3 above)
Comment (juancarlospaco) [reply templates/base.tmpl]: I am not trying to get my packages into this project, not trying to get my packages anywhere,
even more if another package that does the same and is better I would not even create it.

I will revert to all white over the weekend, but lots of coders hate all white background, 
specially me with 50" LCD hahaha :smile: I would use Stylish browser plugin then...
:cat:

**Comment by juancarlospaco** [comment]
Thanks for the feedback :grey_exclamation:

File: templates/pkg.tmpl (hunk 1 above)
Comment (juancarlospaco) [reply templates/pkg.tmpl]: AFAIK `<picture>` should fallback to the existent `` anyways...

**Comment by juancarlospaco** [comment]
Ok!, all Feedback implemented on the code, reverted back to fully white.
:cat:

**Comment by juancarlospaco** [comment]
If you got more Feedback is allways welcome... :smile:

**Comment by Yardanico** [comment]
@juancarlospaco you forgot to remove "#import random_font_color" :)

**Comment by juancarlospaco** [comment]
Oh, forgot to commit that :crying_cat_face:

**Comment by juancarlospaco** [comment]
Any update or feedback?. :smiley:

**Pull request closed without merging**